tests/core/templates/test_rid.h

```diff
@@ -140,7 +140,7 @@ TEST_CASE("[RID_Owner] Thread safety") {
 			uint32_t target = (p_step / 2 + 1) * threads.size();
 			sync[buf_idx].fetch_add(1, std::memory_order_relaxed);
 			do {
-				std::this_thread::yield();
+				Thread::yield();
```

Comment:
Seems consistent to provide a platform-specific `yield`, given a platform-specific `Thread` implementation is used.